templates/email/email-activate.txt

```diff
@@ -8,7 +8,6 @@ your signup:
 
     {{ ACTIVATION_LINK }}
 
-Activation Key: {{ ACTIVATION_KEY }}
```

Comment:
there are selenium acct mgmt tests checking the present of numerous email contents e.g. activation key, might need adjust the test cases accordingly.

to run the account mgmt selenium tests

at the base dir of PR local repo, run :

./docker-compose/./docker-compose/run_selenium_tests_local.sh account

and here is the stdout:
==================
====================================================================== FAILURES ======================================================================
______________________________________________________ TestUserAndAppMgmt.testAccountAndAppMgmt ______________________________________________________

self = <apps.integration_tests.selenium_accounts_tests.TestUserAndAppMgmt object at 0xffff86fc5af0>

    def testAccountAndAppMgmt(self):
        step = [0]
        test_name = "create_user_account"
        api_ver = "*"
        self._print_testcase_banner(test_name, api_ver, step[0], self.use_mslsx, True)
        self._play(ACCT_TESTS[test_name], step, api_ver=api_ver)
        self._print_testcase_banner(test_name, api_ver, step[0], self.use_mslsx, False)
        step = [0]
        # activate request with a faked key should give expected error message with an email
        # for further assistance
        test_name = "validate_activation_key_err_msg"
        self._print_testcase_banner(test_name, api_ver, step[0], self.use_mslsx, True)
        self._play(ACCT_TESTS[test_name], step, api_ver=api_ver)
        self._print_testcase_banner(test_name, api_ver, step[0], self.use_mslsx, False)
        # now activate the user account
>       activation_key = self._validate_email_content(USER_ACCT_ACTIVATION_EMAIL_SUBJ, USER_ACCT_ACTIVATION_KEY_PREFIX)

apps/integration_tests/selenium_accounts_tests.py:27: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <apps.integration_tests.selenium_accounts_tests.TestUserAndAppMgmt object at 0xffff86fc5af0>
subj_line = 'Subject: Verify Your Blue Button 2.0 Developer Sandbox Account', key_line_prefix = 'Activation Key: ', kwargs = {}
f = <_io.TextIOWrapper name='./docker-compose/tmp/bb2_email_to_stdout.log' mode='r' encoding='utf-8'>, log_records = [], email_subj_cnt = 1
key_cnt = 0, ak = None, r = '[28/Jun/2024 15:42:37] "GET /static/legacy-preserved.css HTTP/1.1" 304 0\n'

    def _validate_email_content(self, subj_line, key_line_prefix, **kwargs):
        with open(LOG_FILE, 'r') as f:
            log_records = f.readlines()
            email_subj_cnt = 0
            key_cnt = 0
            ak = None
            while log_records:
                r = log_records.pop(0)
                if r.startswith(subj_line):
                    # print("SUBJ: {}".format(r))
                    email_subj_cnt += 1
                elif key_line_prefix is not None and r.startswith(key_line_prefix):
                    # print("KEY: {}".format(r))
                    ak = r.split(key_line_prefix)[1]
                    key_cnt += 1
                else:
                    pass
                    # print("NOT COUNTED: {}".format(r))
            # assert one and only one expected email (subj line) found
            # if key_line_prefix is not None - need to extract activation key
            assert email_subj_cnt == 1
            if key_line_prefix is not None:
>               assert key_cnt == 1
E               AssertionError

apps/integration_tests/selenium_generic.py:110: AssertionError
--------------------------------------------------------------- Captured stdout setup ----------------------------------------------------------------
set driver_ready=True
use_mslsx=true,  hostname_url=http://bb2slsx:8000
---------------------------------------------------------------- Captured stdout call ----------------------------------------------------------------

******************************************************************
** START TEST: create_user_account, API: *, STEP: 0, Mock SLS
******************************************************************

1:Load BB2 Landing Page ...:
2:Click link 'Signup' to create user account:
3:Type in First Name...:
4:Type in Last Name...:
5:Type in EMAIL...:
6:Type in Organization Name...:
7:Type in Password...:
8:Type in Password again to confirm...:
9:Click link submit button:
10:Check new user login prompt message shown up...:
11:Type in user name (email)...:
12:Type in Password...:
13:Click login button:
14:Check login without activation error message present...:
15:Sleep seconds...:

******************************************************************
** END TEST: create_user_account, API: *, STEP: 15, Mock SLS
******************************************************************


******************************************************************
** START TEST: validate_activation_key_err_msg, API: *, STEP: 0, Mock SLS
******************************************************************

1:Send activation request (with bad activation key)...:
2:Check error message present (There may be an issue with your account. Contact us at..):
3:Sleep seconds...:

******************************************************************
** END TEST: validate_activation_key_err_msg, API: *, STEP: 3, Mock SLS
******************************************************************

============================================================== short test summary info ===============================================================
FAILED apps/integration_tests/selenium_accounts_tests.py::TestUserAndAppMgmt::testAccountAndAppMgmt - AssertionError
================================================================= 1 failed in 35.94s =================================================================

Reply:
done

Reply:
awesome, re-tested and passed:
![image](https://github.com/CMSgov/bluebutton-web-server/assets/135686833/43539a15-e969-43df-af60-489d65988598)